src/ChickenAPI/Game/Components/FamilyComponent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using ChickenAPI.ECS.Components;
+using ChickenAPI.ECS.Entities;
+
+namespace ChickenAPI.Game.Components
+{
+    public class FamilyComponent : IComponent
+    {
+        public FamilyComponent(IEntity entity) => Entity = entity;
+
+        public long FamilyId { get; set; }
+        public byte FamilyLevel { get; set; }
+        public short Players { get; set; }
```

Comment:
Why Players property is represented as short type ?

Reply:
Right, a byte should be enough, will change it asap

Reply:
Should be renamed as "FamilyMemberNumber" or like
And should add property FamilyMemberNumberMax too